File: src/Neuralm.Presentation.Web/src/messages/requests/GetTrainingRoomRequest.ts
import Request from './Request';

/**
 * Represents the GetTrainingRoomRequest class.
 */
export default class GetTrainingRoomRequest extends Request {
  public getId: string;

  /**
   * Initializes the GetTrainingRoomRequest class.
   * @param getId The get id.
   */
  constructor(getId: string) {
    super();
    this.getId = getId;
  }
}

File: src/Neuralm.Services/Neuralm.Services.TrainingRoomService/Neuralm.Services.TrainingRoomService.Messages/GetTrainingRoomRequest.cs
using System;
using Neuralm.Services.Common.Messages;
using Neuralm.Services.Common.Messages.Abstractions;
using Neuralm.Services.Common.Messages.Interfaces;

namespace Neuralm.Services.TrainingRoomService.Messages
{
    /// <summary>
    /// Represents the <see cref="GetTrainingRoomRequest"/> class.
    /// </summary>
    [Message("Get", "/", typeof(GetTrainingRoomResponse))]
    public class GetTrainingRoomRequest : Request, IGetRequest
    {
        /// <inheritdoc cref="IGetRequest.GetId"/>
        public Guid GetId { get; set; }
    }
}